feature: Organization-level coding standard CY-4894 (#823)
codacy/docs: docs/organizations/copying-code-patterns-between-repositories.md

For example, when adding new repositories on Codacy you can copy the tool and pattern configurations from an existing repository that's already configured, and then tweak and adapt the settings for your new repositories.

> !!! tip

Reviewer: @prcr Love this 👍

>     To ensure that multiple repositories consistently follow the same global tool and code pattern configurations, [use an organization coding standard](../organizations/using-a-coding-standard.md) instead.

Reviewer 2: ```suggestion
    To ensure that multiple repositories consistently follow the same tool and code pattern configurations throughout your organization, [use a coding standard](../organizations/using-a-coding-standard.md) instead.
```

I had a thing with "global" first, seemed too absolute. But scoping it to the organization conceptually there, it was redundant at the end. What do you think?

Author: I really like this suggestion, it even makes the sentence simpler! :ok_hand:
